test/src/loader/osm_init.c

```diff
@@ -33,6 +34,16 @@ void
 _PG_init(void)
 {
 	elog(WARNING, "OSM-%s _PG_init", OSM_VERSION_MOD);
+	void *osm_lock_pointer = (LWLock **) find_rendezvous_variable(RENDEZVOUS_OSM_PARALLEL_LWLOCK);
+	if (osm_lock_pointer != NULL)
+	{
+		elog(WARNING, "got lwlock osm lock");
+	}
+	else
+	{
+		elog(WARNING, "NO lwlock osm lock");
```

Comment:
the current unit tests will never reach this branch... is this correct?

Reply:
Yes, that's correct.